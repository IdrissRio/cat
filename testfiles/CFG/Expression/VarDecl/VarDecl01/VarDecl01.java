public class VarDecl01 {
  void m() { int a = 0; }
}