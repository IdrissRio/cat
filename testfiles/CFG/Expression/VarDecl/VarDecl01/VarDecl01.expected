NodeName: Entry[15] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: Entry[7] Successors: [Exit[8]]
NodeName: Exit[8] Successors: []
NodeName: Entry[29] Successors: [IntegerLiteral[46]]
NodeName: IntegerLiteral[46] Successors: [VariableDeclarator[43]]
NodeName: VariableDeclarator[43] Successors: [Exit[36]]
NodeName: Exit[36] Successors: []

NodeName: Exit[16] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: Exit[8] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
NodeName: Exit[36] Predecessor: [VariableDeclarator[43]]
NodeName: VariableDeclarator[43] Predecessor: [IntegerLiteral[46]]
NodeName: IntegerLiteral[46] Predecessor: [Entry[29]]
NodeName: Entry[29] Predecessor: []
